expected/cnpj_1.out

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE empresa(cnpj cnpj);
+\d empresa
+             Table "public.empresa"
+ Column | Type | Collation | Nullable | Default 
+--------+------+-----------+----------+---------
+ cnpj   | cnpj |           |          | 
+
+INSERT INTO empresa VALUES (cnpj '00000000000000');
+SELECT * FROM empresa;
+        cnpj        
+--------------------
+ 00.000.000/0000-00
+(1 row)
+
+DELETE FROM empresa;
+INSERT INTO empresa VALUES (cnpj '33000167066129');
+SELECT * FROM empresa;
+        cnpj        
+--------------------
+ 33.000.167/0661-29
+(1 row)
+
+SELECT cnpj '00000000000001';
+ERROR:  invalid CNPJ
+LINE 1: SELECT cnpj '00000000000001';
+                    ^
+DETAIL:  Invalid check digit for the CNPJ. (00 != 01)
+SELECT cnpj '100000000000000';
+ERROR:  invalid CNPJ
+LINE 1: SELECT cnpj '100000000000000';
+                    ^
+DETAIL:  CNPJ should be less than 99999999999999.
+SELECT cnpj '33.000.167/0661-29';
+        cnpj        
+--------------------
+ 33.000.167/0661-29
+(1 row)
+
+SELECT cnpj '33000.167/0661-29';
+ERROR:  invalid input syntax for integer: "33000.167/0661-29"
+LINE 1: SELECT cnpj '33000.167/0661-29';
+                    ^
+SELECT CNPJ '00000000000000' = CNPJ '00.000.000/0000-00';
+ ?column? 
+----------
+ t
+(1 row)
+
+SELECT CNPJ '00000000000000' = CNPJ '00.001.180/0001-26';
+ ?column? 
+----------
+ f
+(1 row)
+
+SELECT CNPJ '00000000000000' < CNPJ '00.001.180/0001-26';
+ ?column? 
+----------
+ t
+(1 row)
+
```

Comment:
This file should be removed.

Reply:
This file is used by pg_regress in postgres 11, and cnpj.out to 12+. This is necessary right now due different error messages in postgres (ERROR:  invalid input syntax for integer: "33000.167/0661-29" vs. ERROR:  invalid input syntax for type bigint: "33000.167/0661-29") coming from postgres.

If there is a better way to do this, let me know.

Reply:
Ok, I found the documentation: https://github.com/postgres/postgres/blob/master/src/test/regress/pg_regress.c#L1327

the _1 is an alternative expectfile